lang/java/pom.xml

```diff
@@ -347,7 +347,6 @@
                       <pluginExecutionFilter>
                         <groupId>org.apache.maven.plugins</groupId>
                         <artifactId>maven-plugin-plugin</artifactId>
-                        <versionRange>[${plugin-plugin.version},)</versionRange>
```

Comment:
Huh .. is this even defined someplace?  Glad to get rid of it, I guess!

Reply:
No, it wasn't! that's why it was looking an older version and breaking the build on Java 17